EIPS/eip-micropayments_standard_for_nfts_and_multi_tokens.md

```diff
@@ -0,0 +1,318 @@
+
+To take the music use case for example. Traditionally since the industry transitioned from audio distributed on physical medium such as CDs, to an online digital distribution model via streaming, the music industry has been controlled by oligopolies that served to help in the transition. They operate a fixed subscription model and from that they set the amount of royalty distribution to content creators; such as the singers, musicians etc. Using tip tokens represent an additional way for fans of music to reward the content creators. Each song or track is represented by an NFT and fans are able to tip the song (hence the NFT) that they like, and in turn the content creators of the NFT are able to receive the ERC20 rewards that the tips were bought for. A fan led music industry with decentralisation and tokenisation is expected to bring new revenue, and bring fans and content creators closer together.
+
+Across the board in other industries a similar ethos can be applied where third party controllers move to a more facilitating role rather than a monetary controlling role that exists today.
+
+### Guaranteed audit trail
+
+As the Ethereum ecosystem continues to grow, many dapps are relying on traditional databases and explorer API services to retrieve and categorize data. The ERC-xxxx standard guarantees that event logs emitted by the smart contract MUST provide enough data to create an accurate record of all current tip token and ERC20 reward balances. A database or explorer can provide indexed and categorized searches of every tip token and reward sent to NFT holders from the events emitted by any tip token contract that implements this standard. Thus, the state of the tip token contract can be reconstructed from the events emitted alone.
+
+## Backwards Compatibility
+
+A tip token contract can be fully compatible with ERC20 specification and inherit some functions such as transfer if the tokens are allowed to be sent directly to other users. Note that balanceOf has been adopted and MUST be the number of tips held by a user's address. If inheriting from, for example, OpenZeppelin's implementation of ERC20 token then their contract is responsible for maintaining the balance of tip token. Therefore, tip token balanceOf function SHOULD simply directly call the parent (super) contract's balanceOf function.
+
+What hasn't been carried over to tip token standard, is the ability for a spender of other users' tips. For the moment, this standard does not foresee a need for this.
+
+This EIP does not stress a need for tip token secondary markets or other use cases where identifying the tip token type with names rather than addresses might be useful, so these functions were left out of the ITipToken interface and is the remit for implementers.
+
+## Reference Implementation
+
+A [reference/example implementation and tests](https://github.com/julesl23/eip_msnmt) in the GitHub repository that can use be used to assist in implementing this specification.
+
```

Comment:
```suggestion
```
External links should not be included in EIPs.  Either include an inline reference implementation, or files in `../assets/eip-4393/x.sol`.  Alternatively, this is an optional section and can simply be left out if that makes the most sense.

Reply:
> External links should not be included in EIPs. Either include an inline reference implementation, or files in `../assets/eip-4393/x.sol`. Alternatively, this is an optional section and can simply be left out if that makes the most sense.

All sorted in the latest commit. Let me know if there is anything else I need to do.